docker/k8s/README.md

```diff
@@ -27,14 +32,29 @@ in SGX-SIM and SGX-HW mode.
     kubectl create -f lmdb.yaml
     ```
 2. Start `EnclaveManager` (to manage Intel SGX enclaves)  
-   For Intel SGX simulator mode, use the below command
-    ```bash
-    kubectl create -f enclave-manager-deployment.yaml
-    ```
-   For Intel SGX hardware mode, use the below command
-    ```bash
-    kubectl create -f sgx-enclave-manager-deployment.yaml
-    ```
+   Singleton Mode:
```

Comment:
Change this line to the following, to benefit new users not familiar with Singleton Mode:

`Singleton Mode (the same worker handles both keys and workloads):`